src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -120,7 +120,7 @@ public void crossCheckMaximumVitalsValidity() {
 		patientCaptureVitalsPage.setPulseField("78");
 		patientCaptureVitalsPage.setRespiratoryField("1004");
 		assertThat(patientCaptureVitalsPage.getValidationErrors(),
-				hasItem("Maximum: 999"));
+				hasItem("Maximum: 99"));
```

Comment:
Is this change necessary?

Reply:
This is breaking my test workflow ,so I decided to fix it in here

Reply:
> This is breaking my test workflow ,so I decided to fix it in here

@sherrif10 did you get a chance to have a look  at  [RATEST-232](https://issues.openmrs.org/browse/RATEST-232)?

Reply:
> did you get a chance to have a look at [RATEST-232](https://issues.openmrs.org/browse/RATEST-232)?

Sure , but still I could resolve it from here since it was a simple fix that make the test to pass